.github/wiki/Bitrise.md

```diff
@@ -21,7 +21,7 @@ Out of the box, the Bitrise Template has the following workflows and steps:
 | test                    | Create or Update a PR   |
 | deploy_staging          | Push branch `develop`   |
 | deploy_release_firebase | Push branch `release/*` |
-| deploy_app_store        | Push branch `master`    |
+| deploy_app_store        | Push branch `master`/`main`    |
```

Comment:
Optional: the indentation here on the `|` character looks a bit weird and can be reformatted for better alignment 😁